wadsrc/static/zscript/ui/statusbar/statusbar.zs

```diff
@@ -1050,17 +1050,17 @@ class BaseStatusBar native ui
 			for(int i = 0; i < 4; i++) Clip[i] += border;
 
 			//Draw the whole foreground
-			DrawTexture(ontex, position, flags | DI_ITEM_LEFT_TOP);
+			DrawTexture(ontex, position, flags | DI_ITEM_LEFT_TOP, alpha);
 			SetClipRect(position.X + Clip[0], position.Y + Clip[1], texsize.X - Clip[0] - Clip[2], texsize.Y - Clip[1] - Clip[3], flags);
 		}
 		
 		if (offtex.IsValid() && TexMan.GetScaledSize(offtex) == texsize) DrawTexture(offtex, position, flags | DI_ITEM_LEFT_TOP);
```

Comment:
Is it intentional to do not apply alpha to `offtex`?

Reply:
Woops, I accidentally missed that. Will fix